src/pages/steps/CalendarStep/CalendarStep.tsx

```diff
@@ -253,8 +256,27 @@ const CalendarStep = ({
     handleChooseFixedDays();
   }, [scope, isIdle, handleChooseFixedDays]);
 
+  // scroll to calendar step after theme has been selected
+  useEffect(() => {
+    if (!scope || !type.id) return;
+    if (offerId) return;
+
+    const yOffset = -60;
+    const main = document.getElementsByTagName('main')[0];
+    const calendarContainer = document.getElementById(
+      'calendar-step-container',
+    );
+    const scrollToPosition =
+      calendarContainer.getBoundingClientRect().top +
+      window.pageYOffset +
+      yOffset;
+
+    main.scrollTo({ top: scrollToPosition, behavior: 'smooth' });
```

Comment:
I think we can do `calendarContainer.scrollIntoView({behavior: 'smooth'})` to avoid the manual calculations

Reply:
Yes I tried that, the problem is that the title is not visible in that case since the title is not inside the calendar container...

Reply:
That makes sense but in that case could we scroll to the title? I'm worried about the offset calculations being brittle since I've had bad experiences with those in the past depending on browser and platform although it's likely better now I suppose

Reply:
Finally I tried again with the scrollIntoView and seems to work best with the `block` param  set to `center`.
https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView